extensions/ql-vscode/src/remote-queries/export-results.ts

```diff
@@ -100,6 +100,20 @@ async function exportResultsToGist(
   }
 }
 
+/**
+ * Builds Gist description
+ * Ex: Empty Block (Go) x results (y repositories)
+ */
+export const buildGistDescription = (query: RemoteQuery, analysesResults: AnalysisResults[]) => {
```

Comment:
Thanks Charis, I've rejigged the test to check the interface method instead of the `buildGistDescription` method we introduced. Let me know what you think.